tools/scripts/docs_modm_io_index.html.in

```diff
@@ -60,7 +60,6 @@ input[type=submit] {
     top: 100%;
     left: 0;
     right: 0;
-    text-transform: uppercase;
```

Comment:
I spend like 2mins trying to undo CAPLOCK until I realized that this was CSS ;-P

Reply:
IMHO this is necessary for the case-insensitive search to function correctly.

Reply:
No, you already thought about this and did `.toLowerCase()` on both the input and the device list in the function. I left the device upper case though, just made the boards mixed cases as their official names.